tests/nix_tests/network.py

```diff
@@ -109,6 +109,31 @@ def setup_evmos(path, base_port, long_timeout_commit=False):
     yield from setup_custom_evmos(path, base_port, cfg)
 
 
+# for memiavl need to create the data/snapshots dir
+# for the nodes
+def create_snapshots_dir(path, base_port, config):
+    data_snapshots_dir = path / "evmos_9000-1" / "node0" / "data" / "snapshots"
+    os.makedirs(data_snapshots_dir, exist_ok=True)
+    data_snapshots_dir = path / "evmos_9000-1" / "node1" / "data" / "snapshots"
+    os.makedirs(data_snapshots_dir, exist_ok=True)
+
+
+# setup_evmos_rocksdb is evmos chain compiled with rocksdb
+# and configured to use memIAVL + versionBD
+def setup_evmos_rocksdb(path, base_port, long_timeout_commit=False):
```

Comment:
nit: this comment can be added as a docstring
```suggestion
def setup_evmos_rocksdb(path, base_port, long_timeout_commit=False):
    """
    setup_evmos_rocksdb returns an Evmos chain compiled with RocksDB
    and configured to use memIAVL + versionDB.
    """
```